src/transformers/models/egt/collating_egt.py

```diff
@@ -0,0 +1,109 @@
+from typing import Any, Dict, List, Mapping
+
+import numpy as np
+import torch
+
+from ...utils import is_dgl_available, requires_backends
+
+
+if is_dgl_available():
+    import dgl
```

Comment:
Hi, we try to reduce the number of dependencies needed for transformers, please import the related functions in a specific file